docs/var.tfvars-doc.md

```diff
@@ -366,6 +366,16 @@ upgrade_delay_time = "600"
 One of the two varaibles `upgrade_image` or `upgrade_version` is required for upgrading the cluster.
 `upgrade_image` having higher precedence than `upgrade_version`.
 
+
+The following variables are specific to performing EUS upgrades.
+
+```
+#eus_upgrade_version        = ""
+#eus_upgrade_channel        = ""  #(stable-4.x, fast-4.x, candidate-4.x, eus-4.x) eg. stable-4.11
+#eus_upgrade_image          = ""  #(quay.io/openshift-release-dev/ocp-release@sha256:xyz..)
+#eus_upstream               = ""  #https://ppc64le.ocp.releases.ci.openshift.org/graph
```

Comment:
nit: Could you please add the example values inside the quotes instead of comments for showing the correct usage.